### DISCUSSION TITLE
[감사인사] 개발자님 정말 감사합니다

### DISCUSSION THREAD

**Discussion by jin0906** [changdoc/naver-vaccine-macro#132] [General]
8월 말에 출국 예정이였는데 덕분에 2차 까지 맞을수 있어서 자가격리 면제 받고 문제 없게 되었습니다.
정말 감사합니다.

P.S 중국쪽에서 무단으로 가져간거 같은데 확인해보시면 좋을거 같습니다. 링크 남깁니다
https://www.extfans.com/productivity/alfhbmpnlhcpcjjaacapcdnggegicepl

**Comment by changdoc** [comment]
제보 감사합니다. 크롬 스토어를 크롤링해서 수집하는 사이트인가보네요. 크게 문제 없을 것 같습니다.
버전도 예전꺼네요.
